lib/model/autocomplete.dart

```diff
@@ -187,7 +187,7 @@ class MentionAutocompleteView extends ChangeNotifier {
     required PerAccountStore store,
     required Narrow narrow,
   }) {
-    assert(narrow is! CombinedFeedNarrow);
+    assert(narrow is SendableNarrow);
```

Comment:
This is too strong — this class should work on a stream narrow, too. The point is it should work on any narrow where we offer a compose box.

(Do we have an existing test that fails with this change? If not, a good small side fix would be to add one.)

For purposes of this PR, we can just add one more case to this assert, and probably a one-line comment explaining why it's here. There might be a refactor to do if we start prolifierating to three or four such cases.

Reply:
We do have test cases that fail this, which pass `StreamNarrow`s.